InvenTree/email.py

```diff
@@ -0,0 +1,79 @@
+"""Code for managing email functionality in InvenTree."""
+
+import logging
+
+from django.conf import settings
+from django.core import mail as django_mail
+
+import InvenTree.ready
+import InvenTree.tasks
+
+logger = logging.getLogger('inventree')
+
+
+def is_email_configured():
+    """Check if email backend is configured.
+
+    NOTE: This does not check if the configuration is valid!
+    """
+    configured = True
+
+    if InvenTree.ready.isInTestMode():
+        return False
+
+    if InvenTree.ready.isImportingData():
+        return False
+
+    if not settings.EMAIL_HOST:
+        configured = False
+
+        # Display warning unless in test mode
+        if not settings.TESTING:  # pragma: no cover
+            logger.debug("EMAIL_HOST is not configured")
+
+    # Display warning unless in test mode
+    if not settings.TESTING:  # pragma: no cover
+        logger.debug("EMAIL_HOST_USER is not configured")
+
+    # Display warning unless in test mode
+    if not settings.TESTING:  # pragma: no cover
```

Comment:
Question: Where is the check for user or password?

Reply:
https://github.com/inventree/InvenTree/pull/4492

Reply:
Seems like an error in the initial PR by me. We should check before logging - not error out though

Reply:
@matmair I'm not 100% across what you're suggesting here - want to make a suggested change?